src/main/java/rx/Single.java

```diff
@@ -211,6 +211,21 @@ protected Single(final Observable.OnSubscribe<T> f) {
      */
 
     /**
+     * Casts the success value of the current Single into the target type or signals a
+     * ClassCastException if not compatible.
+     * <dl>
+     * <dt><b>Scheduler:</b></dt>
+     * <dd>{@code cast} does not operate by default on a particular {@link Scheduler}.</dd>
+     * </dl>
+     * @param <R> the target type
+     * @param klass the type token to use for casting the success result from the current Single
+     * @return the new Single instance
+     */
+    public final <R> Single<R> cast(final Class<R> klass) {
```

Comment:
@JakeWharton I was following the example of [Observable.cast()](https://github.com/ReactiveX/RxJava/blob/1.x/src/main/java/rx/Observable.java#L5036).  Would you like me to remove `final` from here or shall I leave it to remain consistent?